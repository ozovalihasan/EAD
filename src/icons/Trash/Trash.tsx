import { memo } from "react";

export const Trash = memo(() => (
  <svg
    xmlns="http://www.w3.org/2000/svg"
    viewBox="0 0 200 200"
    className="fill-transparent"
  >
    <path
      style={{strokeLinecap:"round", strokeLinejoin:"round"}}
      d="M 35,50 50,180 H 150 L 165,50"
    />
    <path
      style={{strokeLinecap:"round", strokeLinejoin:"round"}}
      d="m 20,50 160,0"
    />
    <path
      style={{strokeLinecap:"round", strokeLinejoin:"round"}}
      d="M 70,40 80,20 h 40 l 10,20"
    />
  </svg>
))