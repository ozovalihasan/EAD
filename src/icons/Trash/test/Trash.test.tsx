import { Trash } from '../Trash';
import { render } from '@testing-library/react';

let renderReadyComponent: JSX.Element;

beforeEach(() => {
  renderReadyComponent = (
    <Trash></Trash>
  );
});

describe('<Trash />', () => {
  it('renders correctly', () => {
    const renderedContainer = render(renderReadyComponent );
    expect(renderedContainer).toMatchSnapshot();
  });
  
});