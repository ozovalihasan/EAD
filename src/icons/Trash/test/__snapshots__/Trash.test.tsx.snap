// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Trash /> renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        class="fill-transparent stroke-first-400 dark:stroke-first-500"
        viewBox="0 0 200 200"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M 35,50 50,180 H 150 L 165,50"
          style="stroke-linecap: round; stroke-linejoin: round;"
        />
        <path
          d="m 20,50 160,0"
          style="stroke-linecap: round; stroke-linejoin: round;"
        />
        <path
          d="M 70,40 80,20 h 40 l 10,20"
          style="stroke-linecap: round; stroke-linejoin: round;"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      class="fill-transparent stroke-first-400 dark:stroke-first-500"
      viewBox="0 0 200 200"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M 35,50 50,180 H 150 L 165,50"
        style="stroke-linecap: round; stroke-linejoin: round;"
      />
      <path
        d="m 20,50 160,0"
        style="stroke-linecap: round; stroke-linejoin: round;"
      />
      <path
        d="M 70,40 80,20 h 40 l 10,20"
        style="stroke-linecap: round; stroke-linejoin: round;"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
